cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -23,6 +23,16 @@ toolbox:
   tolerations: []
   affinity: {}
 
+ingress: {}
```

Comment:
How about moving the optional sections after the required sections? For example, ingress would come later than cephClusterSpec.

Reply:
I wasn't really sure where to put this, I've put it after the cephClusterSpec for now, but there are several other optional sections above the cephClusterSpec section.